sahi/models/base.py

```diff
@@ -49,6 +50,8 @@ def __init__(
         self.config_path = config_path
         self.model = None
         self.device = device
+        if ".xml" == self.model_path[-4:]:
```

Comment:
Please don't change the base class, move these operations to Yolov8OpenvinoDetectionModel class